stock_quantity_history_location/static/src/components/inventory_report/inventory_report.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module */
+
+import session from "web.session";
+import {patch} from "@web/core/utils/patch";
+import {InventoryReportListController} from "@stock/views/list/inventory_report_list_controller";
+const {onWillStart} = owl;
+
+patch(InventoryReportListController.prototype, "inventory_report_list_controller", {
+    setup() {
+        this._super(...arguments);
```

Comment:
```suggestion
        super.setup();
```

If it is a component, you need this super